spec/lib/data_update_scripts/re_index_feed_content_and_users_to_elasticsearch_spec.rb

```diff
@@ -8,27 +7,49 @@
     Search::User.refresh_index
   end
 
-  it "indexes feed content(articles, comments, podcast episodes) and users to Elasticsearch" do
+  it "indexes feed(articles) to Elasticsearch" do
     article = create(:article)
-    podcast_episode = create(:podcast_episode)
-    comment = create(:comment)
-    user = create(:user)
 
     expect { article.elasticsearch_doc }.to raise_error(Search::Errors::Transport::NotFound)
-    expect { podcast_episode.elasticsearch_doc }.to raise_error(Search::Errors::Transport::NotFound)
-    expect { comment.elasticsearch_doc }.to raise_error(Search::Errors::Transport::NotFound)
-    expect { user.elasticsearch_doc }.to raise_error(Search::Errors::Transport::NotFound)
 
     sidekiq_perform_enqueued_jobs { described_class.new.run }
+
     expect(article.elasticsearch_doc).not_to be_nil
+    expect(article.elasticsearch_doc["_source"].keys).to include("public_reactions_count")
+  end
+
+  it "indexes feed(podcast episodes) to Elasticsearch" do
+    podcast_episode = create(:podcast_episode)
+
+    expect { podcast_episode.elasticsearch_doc }.to raise_error(Search::Errors::Transport::NotFound)
+
+    sidekiq_perform_enqueued_jobs { described_class.new.run }
+
     expect(podcast_episode.elasticsearch_doc).not_to be_nil
+    expect(podcast_episode.elasticsearch_doc["_source"].keys).to include("public_reactions_count")
+  end
+
+  it "indexes feed(comments) to Elasticsearch" do
+    comment = create(:comment)
+
+    expect { comment.elasticsearch_doc }.to raise_error(Search::Errors::Transport::NotFound)
+
+    sidekiq_perform_enqueued_jobs { described_class.new.run }
+
     expect(comment.elasticsearch_doc).not_to be_nil
+
+    expect(comment.elasticsearch_doc["_source"].keys).to include("public_reactions_count")
+  end
+
+  it "indexes users to Elasticsearch" do
+    user = create(:user)
+
+    expect { user.elasticsearch_doc }.to raise_error(Search::Errors::Transport::NotFound)
+
+    sidekiq_perform_enqueued_jobs { described_class.new.run }
+
     expect(user.elasticsearch_doc).not_to be_nil
 
-    expect(article.elasticsearch_doc["_source"].keys).to include "public_reactions_count"
```

Comment:
Since this is going to increase the run time of this spec, could we instead use `aggregate_failures` here to keep all the Elasticsearch refreshing etc to a single time?

Reply:
Got it. I'm going to put it back as it were then and add `aggregate_failures`